### DISCUSSION TITLE
Cookie banner

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#5970] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Hi

I've installed adguard home with tls://dns.adguard-dns.com asvDNS upstream server and
94.140.14.14
94.140.15.15
2a10:50c0::ad1:ff
2a10:50c0::ad2:ff
As starter dns

I'd Like to know how ça i strike the cookie banner on every website

Thanks

Jerome L

### Proposed solution

-

### Alternatives considered and additional information

-

**Comment by ainar-g** [comment]
Hello.  Cookie banners are a part of a site's content, while AdGuard Home is a DNS server, and so cannot directly filter a site's content.  See [this entry in the F.A.Q.](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#limitations)

You can still use a browser extension to remove banners.
